README.md

```diff
@@ -30,34 +30,70 @@ $ pip install .
 
 ## Set up a .gitreview.
 
-Before using this tool, you need to create a .gitreview file in the root
-directory of your project. It needs to contain info like GitLab project ID and
-a private token that can be used to access the GitLab repo and the GitLab API.
-Current optional configuration flags include `target_branch` and
-`remove_source_branch`. `target_branch` represents the target branch that you
-want the MRs to eventually merge into. By default, `target_branch` is `master`.
-`remove_source_branch` can be used to delete the source branch of an MR once
-it's merged. By default, this is set to `True`.  The following shows an example
-of `.gitreview`:
+Before using this tool, you need to create a `.gitreview` file in the root
+directory of your project.
+
+It must contain:
+
+* `host`: The base URL of the GitLab server.
+* `project_id`: The ID of your project on your GitLab host.
+
+It may optionally contain:
+
+* `target_branch`: the target branch that you want the MRs to eventually merge into. By default, `target_branch` is `master`.
+* `remove_source_branch`: Boolean value, indicating whether the source branch of an MR should be deleted once it's merged. By default, `remove_source_branch` is `True`.
+
+Example `.gitreview`:
 
 ```ini
 [origin]
 host=https://gitlab.example.com
 project_id=1234
-private_token=[your-private-token]
 target_branch=master
 remove_source_branch=True
 ```
 
+## Set up a private token.
+
+GerritLab requires a GitLab private token with api access.
+
+You have the option to store it either in:
+
+1. `git config`
+2. [git credential helper](https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage)
+
+### Option 1: Store in git config
 
-The `private_token` can alternatively also be stored in your Git config:
+To store in your git config:
 
 ```console
 $ git config --global gerritlab.private-token "[your-private-token]"
 # OR
 $ git config --local gerritlab.private-token "[your-private-token]"
 ```
 
+### Option 2: Use a git credential store
+
+A more secure option is to use a **git credential store**. This option enables you
+to store a token in your operating system's password store.
+
+GerritLab stores the private token in your credential manager using the following configuration:
+
+```
+protocol=gerritlab
```

Comment:
Instead of including the protocol details, I suggest just describing how the credentials must be named/identified/titled in your password, with an example.

Reply:
Good call, attempted that in latest revision.